src/components/Staking/StakingConsiderations/index.tsx

```diff
@@ -75,9 +75,18 @@ const StakingConsiderations = ({ page }: StakingConsiderationsProps) => {
     pageData,
     activeIndex,
   } = useStakingConsiderations({ page })
+  
+  const activeStyles = {
+    bg: "var(--eth-colors-offBackground)",
+    color: "var(--eth-colors-chakra-body-text)",
+    fontWeight: "bold",
+    display: "table",
+    width: "100%",
+    transition: 'font-weight 0.3s ease, color 0.3s ease'
```

Comment:
@iepn On the font-weight transition, with #12501 we removed the use of variable fonts and replaced with static... I don't believe this effect will work without the variable font. (Those changes are in `dev` now if you want to update your branch)

Reply:
I pulled down the latest commit:

In the end, the following styles are effective:

```diff
  const activeStyles = {
    bg: "background.highlight",
    color: "body.base",
    fontWeight: "bold",
-   display: "table",
-    width: "100%",
-    transition: 'font-weight 0.3s ease, color 0.3s ease'
  }
```